### PR TITLE
Translated the document of the Login component.

### PR DESCRIPTION
**Pull request by SimpleFrontend**
#120 Translated the document of the Login component.

### CODE REVIEW

File: src/components/Login/index.md [hunk 1]
```diff
@@ -2,50 +2,49 @@
 title:
   en-US: Login
   zh-CN: Login
-subtitle: 登录
+subtitle: Login
```
Comment (afc163) [inline src/components/Login/index.md]: You should create both `index.en-US.md` and `index.zh-CN.md`

Comment (SimpleFrontend) [reply src/components/Login/index.md]: Thanks. Totally missed that. I've updated it.

File: src/components/Login/demo/basic.md [hunk 2]
```diff
@@ -1,9 +1,12 @@
 ---
 order: 0
 title: Standard Login
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: this line should be deleted

File: src/components/Login/demo/basic.md [hunk 3]
```diff
@@ -26,7 +29,7 @@ class LoginDemo extends React.Component {
         if (!err && (values.username !== 'admin' || values.password !== '888888')) {
           setTimeout(() => {
             this.setState({
-              notice: '账号或密码错误！',
+              notice: 'the combination of username and password is incorrect!',
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: The(should be capitalized)

File: src/components/Login/demo/basic.md [hunk 4]
```diff
@@ -1,9 +1,12 @@
 ---
 order: 0
 title: Standard Login
+title:
+  zh-CN: 标准登录
+  en-US: Standard Login
 ---
 
-支持账号密码及手机号登录两种模式。
+Support login with username/password and mobile number.
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: `Support login with account and mobile number` is Ok

File: src/components/Login/demo/basic.md [hunk 5]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
         onTabChange={this.onTabChange}
         onSubmit={this.onSubmit}
       >
-        <Tab key="tab1" tab="账号密码登录">
+        <Tab key="tab1" tab="username/password">
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: username/password -> account

File: src/components/Login/demo/basic.md [hunk 6]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
...
           {
             this.state.notice &&
             <Alert style={{ marginBottom: 24 }} message={this.state.notice} type="error" showIcon closable />
           }
           <UserName name="username" />
           <Password name="password" />
         </Tab>
-        <Tab key="tab2" tab="手机号登录">
+        <Tab key="tab2" tab="mobile">
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: Mobile

File: src/components/Login/demo/basic.md [hunk 7]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
...
           <Mobile name="mobile" />
           <Captcha onGetCaptcha={() => console.log('Get captcha!')} name="captcha" />
         </Tab>
         <div>
-          <Checkbox checked={this.state.autoLogin} onChange={this.changeAutoLogin}>自动登录</Checkbox>
-          <a style={{ float: 'right' }} href="">忘记密码</a>
+          <Checkbox checked={this.state.autoLogin} onChange={this.changeAutoLogin}>keep me logged in</Checkbox>
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: Keep...

File: src/components/Login/demo/basic.md [hunk 8]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
...
+          <a style={{ float: 'right' }} href="">forget password</a>
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: forget password -> Forgot password?

File: src/components/Login/demo/basic.md [hunk 9]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
...
         </div>
-        <Submit>登录</Submit>
+        <Submit>Login</Submit>
         <div>
-          其他登录方式
+          other login methods
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: Other...

File: src/components/Login/demo/basic.md [hunk 10]
```diff
@@ -50,29 +53,29 @@ class LoginDemo extends React.Component {
...
           <span className="icon icon-alipay" />
           <span className="icon icon-taobao" />
           <span className="icon icon-weibo" />
-          <a style={{ float: 'right' }} href="">注册账户</a>
+          <a style={{ float: 'right' }} href="">register</a>
```
Comment (ddcat1115) [inline src/components/Login/demo/basic.md]: Register

File: src/components/Login/index.en-US.md [hunk 11]
```diff
@@ -0,0 +1,48 @@
+---
+title: Login
+subtitle: Login
```
Comment (ddcat1115) [inline src/components/Login/index.en-US.md]: useless line

Comment (ddcat1115) [reply src/components/Login/index.en-US.md]: And `index.zh-CN.md` shloud make some changes, see https://github.com/ant-design/ant-design-pro/pull/123/files#diff-eec8bce4026425cef4b8075981a9d2fd

File: src/components/Login/index.en-US.md [hunk 12]
```diff
@@ -0,0 +1,48 @@
...
+cols: 1
+order: 15
+---
+
+Support multipal common ways of login with built-on controls. You can choose your own combinations and use with your custom controls.
```
Comment (ddcat1115) [inline src/components/Login/index.en-US.md]: multiple

Comment (ddcat1115) [reply src/components/Login/index.en-US.md]: built-in

File: src/components/Login/index.en-US.md [hunk 13]
```diff
@@ -0,0 +1,48 @@
...
+----|------|-----|------
+key | key of the tab | String | -
+tab | displayed text of the tab | ReactNode | -
+
+### Login.UserName
+
+Property | Description | Type | Default
+----|------|-----|------
+name | name of the control, also the key of the submitted data | String | -
+rules | validation rules, same with [option.rules 的规则](getFieldDecorator(id, options)) in Form getFieldDecorator(id, options) | object[] | -
```
Comment (ddcat1115) [inline src/components/Login/index.en-US.md]: chinese

File: src/components/Login/index.en-US.md [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+
+Apart from the above properties, Login.Username also support all properties of antd.Input, together with the default values of basic settings, such as _placeholder_, _size_ and _prefix_. All of these default values can be over-written.
+
+### Login.Password, Login.Mobile, and Login.UserName
```
Comment (ddcat1115) [inline src/components/Login/index.en-US.md]: Login.Password, Login.Mobile are the same as Login.UserName

File: src/components/Login/index.en-US.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+
+### Login.Captcha
+
+Property | Description | Type | Default
+----|------|-----|------
+onGetCaptcha | callback on getting a new Captcha image | () => void | -
```
Comment (ddcat1115) [inline src/components/Login/index.en-US.md]: callback on getting a new captcha ~~image~~

File: src/components/Login/map.js [hunk 16]
```diff
@@ -23,31 +23,31 @@ const map = {
...
+      pattern: /^1\d{10}$/, message: 'Wrong mobile number format!',
     }],
   },
   Captcha: {
     component: Input,
     props: {
       size: 'large',
       prefix: <Icon type="mail" className={styles.prefixIcon} />,
-      placeholder: '验证码',
+      placeholder: 'verification code',
```
Comment (ddcat1115) [inline src/components/Login/map.js]: captcha

File: src/components/Login/map.js [hunk 17]
```diff
@@ -23,31 +23,31 @@ const map = {
...
     },
     rules: [{
-      required: true, message: '请输入验证码！',
+      required: true, message: 'Please enter verification code!',
```
Comment (ddcat1115) [inline src/components/Login/map.js]: Please enter Captcha!

**Comment by ddcat1115** [comment]
@SimpleFrontend  Some changes need to be made 😅

**Comment by SimpleFrontend** [comment]
@ddcat1115 Thanks for the feedback. Updated.

**Pull request merged**